### ISSUE TITLE
CVE Reporting - False positives

### ISSUE THREAD

**Issue by imponimassacri** [OCSInventory-NG/OCSInventory-ocsreports#953]
CVE scan reports some false positives. For example, version php7.3 is reported ad affected by CVE-2007-5653
It would be useful to add a column in which to insert a comment to remember having already checked the vulnerability and having considered it a false positive or a check to hide the vulnerability.

**Comment by gillesdubois** [comment]
Hi,

Quote from the wiki : 
```
VULN_CVESEARCH_ALL : CVE-Search sometimes inventory cve without any software version. If this settings is off, these non version specific CVE won't be processed by OCS.
```

Disabling this option won't get old CVE that doesn't match a specific version.

Regards,
Gilles Dubois.

**Comment by imponimassacri** [comment]
The VULN_CVESEARCH_ALL parameter is present in the wiki, but it is not configurable in the Cve_search configuration page...

**Comment by charleneauger** [comment]
Hi @imponimassacri ,

What is your server version ?

Regards,
Charlene Auger

**Comment by imponimassacri** [comment]
Hi Charlene, the version is 2.7, downloaded from https://ocsinventory-ng.org/?page_id=1548&lang=en

**Comment by charleneauger** [comment]
@imponimassacri , VULN_CVESEARCH_ALL is available on 2.7 nightly version. You can download one of this version [here](http://download.ocsinventory-ng.org/nightly/). With nightly version, you also have a [CVE matching expressions](http://wiki.ocsinventory-ng.org/04.Management-console-and-its-advanced-features/CVE-Search-match-regex/) feature.

Regards,
Charlene
